hasktorch/src/Torch/Static/Native.hs

```diff
@@ -194,47 +189,11 @@ sqrt t = unsafePerformIO $ (cast1 ATen.sqrt_t) t
 tanh :: Tensor dtype shape -> Tensor dtype shape
 tanh t = unsafePerformIO $ (cast1 ATen.tanh_t) t
 
--- |
--- >>> dtype &&& shape $ gt (ones :: Tensor 'D.Float '[2,2]) (zeros :: Tensor 'D.Float '[2,2])
--- (Bool,[2,2])
-gt :: Tensor dtype shape -> Tensor dtype shape -> Tensor 'D.Bool shape
-gt a b = unsafePerformIO $ (cast2 ATen.gt_tt) a b
-
-(>.) = gt
-
-lt :: Tensor dtype shape -> Tensor dtype shape -> Tensor 'D.Bool shape
-lt a b = unsafePerformIO $ (cast2 ATen.lt_tt) a b
-
-(<.) = lt
-
-ge :: Tensor dtype shape -> Tensor dtype shape -> Tensor 'D.Bool shape
-ge a b = unsafePerformIO $ (cast2 ATen.ge_tt) a b
-
-(>=.) = ge
-
-le :: Tensor dtype shape -> Tensor dtype shape -> Tensor 'D.Bool shape
-le a b = unsafePerformIO $ (cast2 ATen.le_tt) a b
-
-(<=.) = le
-
--- |
--- >>> dtype &&& shape $ eq (ones :: Tensor 'D.Float '[2,2]) (zeros :: Tensor 'D.Float '[2,2])
--- (Bool,[2,2])
-eq :: Tensor dtype shape -> Tensor dtype shape -> Tensor 'D.Bool shape
-eq a b = unsafePerformIO $ (cast2 ATen.eq_tt) a b
-
-(==.) = eq
-
-ne :: Tensor dtype shape -> Tensor dtype shape -> Tensor 'D.Bool shape
-ne a b = unsafePerformIO $ (cast2 ATen.ne_tt) a b
-
-(/=.) = ne
-
 -- |
 -- >>> dtype &&& shape $ (toDType (ones :: Tensor 'D.Float '[2,2]) :: Tensor 'D.Double '[2,2])
 -- (Double,[2,2])
 toDType
-  :: forall dtype dtype' shape
+  :: forall dtype' dtype shape
```

Comment:
it's true for type arguments, too:
https://twitter.com/rob_rix/status/1171495487267991555?s=20